mslib/mscolab/_tests/test_chat_manager.py

```diff
@@ -23,35 +23,37 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 """
-
 import requests
 import json
-from werkzeug.urls import url_join
+import sys
+import time
 
+from werkzeug.urls import url_join
+from PyQt5 import QtWidgets
 from mslib.mscolab.models import User, MessageType, Message
-from mslib._tests.constants import MSCOLAB_URL_TEST
 from mslib.mscolab.conf import mscolab_settings
-from mslib.mscolab.server import db, APP, initialize_managers
-from mslib.mscolab.utils import get_recent_pid
 from mslib.mscolab.chat_manager import ChatManager
+from mslib.msui.mscolab import MSSMscolabWindow
+from mslib._tests.utils import mscolab_start_server
+
+
+PORTS = list(range(9321, 9340))
 
 
 class Test_Chat_Manager(object):
     def setup(self):
-        self.sockets = []
-        self.app = APP
-        self.app.config['SQLALCHEMY_DATABASE_URI'] = mscolab_settings.SQLALCHEMY_DB_URI
-        self.app.config['MSCOLAB_DATA_DIR'] = mscolab_settings.MSCOLAB_DATA_DIR
-        self.app, _, _, fm = initialize_managers(self.app)
-        self.fm = fm
+        self.process, self.url, self.app, _, self.cm, self.fm = mscolab_start_server(PORTS)
```

Comment:
that is wanted

Reply:
At the moment I have learned how I can use a fixture in the class this will be moved to conftest. I want only one place where I have to add self.window.show().